src/newSeed/stage3.ts

```diff
@@ -34,12 +34,15 @@ export class Stage3 extends BaseStage {
       this.stageState.verified = false;
```

Comment:
I think we should limit all changes to `this.stageState.verified` so they occur only inside of `validateInputs()`